exercises/brandon_hewer/ex02_oo_basics/src/Circle.cpp

```diff
@@ -0,0 +1,72 @@
+//----------------------------------------------------------------------
+// Includes
+//----------------------------------------------------------------------
+#include "Circle.h"
+
+/*
+ * Constructs a new Circle with the specified radius.
+ *
+ * @param radius	The radius of the circle to construct.
+ *
+ * @throw invalid_argument	Thrown if specified radius is less than zero.
+ */
+Circle::Circle(double radius) {
+	
+	if (radius < 0) {
+		std::ostringstream ss;
+		ss << "Radius of a " << name() << " must be greater than or equal"
+			"to 0";
+		throw ss.str();
```

Comment:
It is very good that you checked :+1: 

You should throw however an appropriate error class, eg.